### ISSUE TITLE
Start/Restart KiwiIRC on reboot

### ISSUE THREAD

**Issue by perthsystem** [prawnsalad/KiwiIRC#658]
I'm using Ubuntu 14.01 anyone suggest a way to to restart Kiwi after a reboot.

I have tried @reboot but realized that it won't start until networking is up and running.

**Comment by prawnsalad** [comment]
You might want to look into Ubuntu services.
http://kvz.io/blog/2009/12/15/run-nodejs-as-a-service-on-ubuntu-karmic/
http://unix.stackexchange.com/questions/84252/how-to-start-a-service-automatically-when-ubuntu-starts

**Comment by perthsystem** [comment]
Thanks.

I actually worked it out.
The problem was that for some reason the filepath to node: was not working from crontab so I added the full path to node in the kiwi script and all is well now. As a matter of interested to anyone else wanting to have kiwi start on reboot, I found that a crontab to 'kiwi start' run every 5 or 10 minutes or so, did the job of starting it at reboot as well making sure it's always running.
